.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,51 @@
+name: '🐛 | Bug Report'
+description: >
+  File a bug report
+title: '[🐛 | Bug]: '
+labels: [bug, triage]
+assignees:
+  - touilleMan
+body:
+  - type: textarea
+    id: what-happened
+    attribute:
+      label: What happened ?
+      description: 'Also tell us, what did you expect to happen ?'
+      placeholder: 'Tell us what you see !'
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: 'What version of our software are you running ?'
+      options:
+        - 2.8.3
```

Comment:
Why not a free field instead ? this is going to be a pain to keep this field up to date.

Reply:
It would be easier to fill for the user but harder to maintain.

I can add a field that say `version not listed` or something